docs/ops/opset.md

```diff
@@ -6,6 +6,7 @@ This topic provides a complete list of available sets of operations supported in
 
 | OpenVINO™ Version | Actual Operations Set            | 
 | :---------------- | :------------------------------- | 
+| 2021.3            | [opset6](opset6.md)   | 
```

Comment:
Please, update the `./docs/doxygen/ie_docs.xml` line 85 as well.

Reply:
Done